insights.md

```diff
@@ -24,10 +24,11 @@ Run Insights lets you filter the shown data per master or group of masters to un
 * It stores the filter setting in your local storage, to persist the filter when you return.
 
 
-1. The initial view will show you insights for all Jenkins Masters that are connected to the ***cloudbees-days*** account. To filter by your Team Master search for your Team Master name in the **Filter by Connected Masters...** search input and select your Team Master from the list <p><img src="img/insights/insights_filter_by_master_input.png" width=800/>
+1. The initial view will show you insights for all Jenkins Masters that are connected to the ***cloudbees-dwjw*** organization. To filter by your Team Master search for your Team Master name in the **Filter by Connected Masters...** search input and select your Team Master from the list <p><img src="img/insights/insights_filter_by_master_input.png" width=850/>
```

Comment:
"connected to the ***cloudbees-dwjw*** organization" - You may want to make this generic as different orgs may be used for different workshops - like we did for CB Days.

Reply:
Thanks - updated as follows: "...connected to your assigned organization configured in CloudBees DevOptics"